src/runtime_src/core/tools/xbutil2/OO_MemRead.cpp

```diff
@@ -132,13 +134,17 @@ OO_MemRead::execute(const SubCmdOptions& _options) const
   XBU::verbose(boost::str(boost::format("Device: %s") % xrt_core::query::pcie_bdf::to_string(xrt_core::device_query<xrt_core::query::pcie_bdf>(device))));
```

Comment:
Shouldn't there be a DRC check to make sure count is greater then 0 as well as for the size being greater then zero.